multiplayer/src/epics/notifyGameDisconnected.ts

```diff
@@ -63,6 +63,16 @@ export function notifyGameDisconnected(reason: GameOverReason | undefined) {
                         icon: "😟",
                     })
                 );
+            case "compile-failed":
+                pxt.tickEvent("mp.compilefailed");
+                return dispatch(
+                    showToast({
+                        type: "error",
+                        text: lf("Game failed to compile"),
```

Comment:
nit: "Unable to compile game" (or something to that effect) might be a slightly less user-blame-y phrasing.

Reply:
"compile" may not be a familiar term for many. How about "There was a problem loading this game"

Reply:
Yeah, I was thinking about that as well, I find this message a little challenging to craft as the problem most likely does come from the game itself, so ideally we would point them in the direction of fixing that so they can progress (vs. just trying to host the same game again or something) -- if the message is ambiguous it might seem like a fault in multiplayer and then they might not end up being able to play or something. I think your wording is enough to drive people but this probably just needs testing to get the right wording?